pkg/cmd/pulumi/state_move.go

```diff
@@ -182,6 +204,9 @@ func (cmd *stateMoveCmd) Run(
 		destSnapshot.Resources = append(destSnapshot.Resources, r)
 	}
 
+	// TODO: conflict on resources that are already on the destination stack
```

Comment:
Is this covered elsewhere/in a stacked PR/issue?

Reply:
Yes, here: https://github.com/pulumi/pulumi/pull/16562